lib/boombox.ex

```diff
@@ -17,12 +21,45 @@ defmodule Boombox do
           | {:audio_channels, Membrane.RawAudio.channels_t()}
         ]
 
+  @typedoc """
+  Some encodings can/must be accompanied with encoding specific parameters:
+    * AAC:
+      - bitrate_mode - has to be provided. Defines how the RTP stream was payloaded and should be depayloaded.
+      - audio_specific_config - has to be provided. Contains crucial information about the stream and has to be obtained from a side channel.
+    * H264:
+      - ppss and spss - optional. picture and sequence parameter sets, could be obtained from a side channel. They contain information about the encoded stream.
+  """
+  @type rtp_encoding_specific_params ::
+          {:AAC, [bitrate_mode: RTP.AAC.Utils.mode(), audio_specific_config: binary()]}
+          | {:H264, [ppss: [binary()], spss: [binary()]]}
```

Comment:
- it would be nice if we could pass binaries encoded in HEX/base64 via cmdline, I think FFmpeg allows it too
- How about H265 and Opus?

Reply:
Maybe we could have some default mapping between cli and elixir apis when it comes to binaries, like base64 <-> binary?

Reply:
yup, good idea, it can be a separate task

Reply:
That's a lie :D https://hexdocs.pm/membrane_rtp_h265_plugin/readme.html